modules/nf-core/fastp/main.nf

```diff
@@ -0,0 +1,98 @@
+process FASTP {
+    tag "$meta.id"
+    label 'process_medium'
```

Comment:
FYI: i found fastp to be the fastest when running on 12 cpus which is also inline with some issue i found on their repo. NOt sure how critical this is for your datasize, but just in case.